src/main/java/salesforce/ui/pages/lightning/worktype/WorkTypeDetails.java

```diff
@@ -107,6 +76,27 @@ public String getTxtField(final String nameTxtBox) {
         return webDriverActions.getTextByXpathLocator(String.format(pathFieldTxt, nameTxtBox));
     }
 
+    /**
+     * Sets value and select in txt of Duration Type, Block Time Before Unit, Block Time After Unit,
+     * Time Frame Start Unit and Time Frame End Unit.
+     *
+     * @param nameTxtBox is name of textBox
+     * @return value of nameTxtBox
+     */
+    public String getUnitsOfTxtField(final String nameTxtBox) {
```

Comment:
The method says it gets a value of a text field but a textbox name is entered.

Reply:
Because its path permit to get txt different txtbox